<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilTest</class>
 <widget class="QWidget" name="UtilTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>310</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>First contour points: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="firstFiducialSelector">
     <property name="toolTip">
      <string>Pick the input to the algorithm.</string>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLMarkupsClosedCurveNode</string>
      </stringlist>
     </property>
     <property name="showChildNodeTypes">
      <bool>false</bool>
     </property>
     <property name="noneEnabled">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>true</bool>
     </property>
     <property name="removeEnabled">
      <bool>true</bool>
     </property>
     <property name="editEnabled">
      <bool>false</bool>
     </property>
     <property name="renameEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Second contour points:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="qMRMLNodeComboBox" name="secondFiducialSelector">
     <property name="toolTip">
      <string>Pick the output to the algorithm.</string>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLMarkupsClosedCurveNode</string>
      </stringlist>
     </property>
     <property name="showChildNodeTypes">
      <bool>false</bool>
     </property>
     <property name="noneEnabled">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>true</bool>
     </property>
     <property name="removeEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Sampling distance (mm):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ctkSliderWidget" name="samplingDistanceSliderWidget">
     <property name="toolTip">
      <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="pageStep">
      <double>2.000000000000000</double>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Enable Screenshots:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="enableScreenshotsFlagCheckBox">
     <property name="toolTip">
      <string>If checked, take screen shots for tutorials. Use Save Data to write them to disk.</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
